src/base.ls

```diff
@@ -1,21 +1,20 @@
 require! {
-  'lodash.assign': assign
   react
 }
 
 
-module.exports = (classnames) -> (element, opts = {}, ...children) ->
+module.exports = (classnames) -> (element, opts, ...children) ->
   if typeof element is \string
     [element, ...classes] = element.split '.'
     element = 'div' if element is ''
 
-  if opts not instanceof Object or opts instanceof Array or opts.$$typeof?
-    children = [opts, ...children]
-    opts = {}
+  opts = if opts not instanceof Object or opts instanceof Array or opts.$$typeof?
```

Comment:
I think I liked this being named cloned-opts to make it easier to tell we weren't morphing input. We have the test for not morphing it so I guess that not too important, though this does read weird for the else case where it ends up being
```
opts = {...opts}
```